src/server/index.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+
+var mongodb = require('mongodb');
+var MongoClient = mongodb.MongoClient;
+var url = 'mongodb://localhost:27017/test';
+
+app.use(bodyParser.json());
+
+app.get('/heartbeat', function (req, res) {
+  let query = {};
+  MongoClient.connect(url, function (err, db) {
+    if (err) {
+      console.log('Unable to connect to the MongoDB server. Error:', err);
+      res.send({"status": "ok", "database": "error"});
+      throw err;
```

Comment:
if you throw the error here it will crash the whole server since you don't catch it anywhere.
i'd remove this line so the server keeps running